### ISSUE TITLE
About Mobile Device Benchmarks

### ISSUE THREAD

**Issue by sailor002** [bes-dev/MobileStyleGAN.pytorch#26]
Hi,

Thanks for the great work. Do you have benchmarks on the performance of the model, like CoreML prediction speed on specific devices? Also, I wonder how prediction times change by the resolution of the model (256, 512, 1024) and how big these models are in size. Thanks...

**Comment by bes-dev** [comment]
@sailor002 hey, thanks for your interest to our project. 
So, we tried to run our pre-trained model (1024x1024) on the mobile devices. For iPhone 12 PRO MAX using CoreML (CPU + GPU + NPU scheduler) MobileStyleGAN works 220ms per image for 1024x1024 resolution (you can found sample video here: https://www.youtube.com/watch?v=CBebSII31uE&list=PLstKhmdpWBtwsvq_27ALmPbf_mBLmk0uI&index=5). But we didn't profile our model for other devices or less image size.

**Comment by sailor002** [comment]
Thanks, @bes-dev 👍
